main.py

```diff
@@ -1,10 +1,67 @@
+import json
+from typing import Optional, Any
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
+def load_players_from_file() -> dict[str, Any]:
+    with open("players.json", "r") as file:
+        players = json.load(file)
+    return players
+
+
+def create_race(race_info: dict[str, Any]) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=race_info["name"],
+        description=race_info["description"]
+    )
+    return race
+
+
+def create_skills(skills_info: list[dict[str, Any]], race: Race) -> None:
+    for skill in skills_info:
+        Skill.objects.get_or_create(
+            name=skill["name"],
+            bonus=skill["bonus"],
+            race=race
+        )
+
+
+def create_guild(guild_info: Optional[dict[str, Any]]) -> Optional[Guild]:
+    if guild_info:
+        guild, _ = Guild.objects.get_or_create(
+            name=guild_info["name"],
+            description=guild_info["description"]
+        )
+    else:
+        guild = None
+    return guild
```

Comment:
```suggestion
def create_guild(guild_info: Optional[dict[str, Any]]) -> Optional[Guild]:
    guild = None
    if guild_info:
        guild, _ = Guild.objects.get_or_create(
            name=guild_info["name"],
            description=guild_info["description"]
        )
    return guild
```
it will be a little cleaner)